tests/FSharp.Compiler.ComponentTests/Scripting/Interactive.fs

```diff
@@ -63,6 +64,15 @@ module Test2 =
         Assert.Equal(typeof<string>, value2.ReflectionType)
         Assert.Equal("Execute - Test2.test2 - 27", value2.ReflectionValue :?> string)
 
+    [<Theory>]
```

Comment:
This looks more like a bcl test case than one interesting to us.

Reply:
Yep, I'll remove it.